CHANGELOG.md

```diff
@@ -73,6 +74,10 @@
 - Added support for `DataFrame.corr`.
 - Added support for `DataFrame.equals` and `Series.equals`.
 - Added support for `DataFrame.reindex` and `Series.reindex`.
+- Added support for `Index.T`.
```

Comment:
actually, @sfc-gh-vbudati when is Index.T added? is it in an older version or 1.20.0

Reply:
I double checked, index.T seems have been added in 1.19.0, can you add it at the correct place?

Reply:
added